addons/mail/static/src/js/main.js

```diff
@@ -90,6 +90,10 @@ Object.assign(env, {
     messagingCreatedPromise: createMessaging(),
     modelManager: new ModelManager(env),
     store,
+    /**
+     * Useful for notifications to apply header title color.
+     */
+    titleColor: '#212529',
```

Comment:
As the `env` is generic, let's go with a more specific name here. I suggest `notificationTitleColor`. (don't forget alphabetical order after rename)